test.sh

```diff
@@ -64,10 +97,107 @@ function run_test_coverage() {
   goveralls -v -coverprofile=gover.coverprofile -service=travis-pro
 }
 
+#
+# Main CLI Parser
+#
+USAGE="$(cat -- <<-EOM
+
+Usage:
+Boulder test suite CLI, intended to be run inside of a Docker container:
+
+  docker-compose run --use-aliases boulder ./$(basename "${0}") [OPTION]...
+
+With no options passed: runs standard battery of tests (lint, unit, and integation)
+    -l, --lints                           Adds lint to the list of tests to run
+    -u, --unit                            Adds unit to the list of tests to run
+    -d, --unit-test-filter <DIRECTORY>    Run unit tests for a specific directory
+    -e, --enable-unit-race                Enables -race flag for all unit tests
+    -n, --config-next                     Changes BOULDER_CONFIG_DIR from test/config to test/config-next
+    -c, --coverage                        Adds coverage to the list of tests to run
+    -i, --integration                     Adds integration to the list of tests to run
+    -s, --start-py                        Adds start (py) to the list of tests to run
```

Comment:
```suggestion
    -s, --start-py                        Adds start.py to the list of tests to run
```

Reply:
I think that this is actually a bad idea. The longopt is supposed to be descriptive, but the help message needs to be accurate. This would suggest that adding `-s` or `--start-py` adds a `start.py` to the `RUN` array.